python/shotgun_model/simple_shotgun_hierarchy_model.py

```diff
@@ -31,7 +31,7 @@ class SimpleShotgunHierarchyModel(ShotgunHierarchyModel):
     can also be subclassed from this class.
     """
 
-    def load_data(self, seed_entity_field, path=None, entity_fields=None):
+    def load_data(self, seed_entity_field, root=None, entity_fields=None):
```

Comment:
@manneohrstrom said we should allow to break this one since it uses an API that is internal (nav_api) and as such is experimental. Do you still think so Manne?